composer.json

```diff
@@ -39,8 +39,8 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.7.1",
         "mikey179/vfsStream": "1.1.0",
-        "phpunit/phpunit": "^5.7",
-        "matthiasnoback/symfony-dependency-injection-test": "~1.0",
+        "phpunit/phpunit": "~6.0",
```

Comment:
Unless targeting 7.0, this should remain at `^5.7|^6.0`, due to 6.0 not supporting PHP 5.6.

Reply:
@MarioBlazek Like in LegacyBridge, could we use tilde here just for clarity?